layouts/accounts.html

```diff
@@ -0,0 +1,9 @@
+<h1 id="account-workflow">Accounts</h1>
```

Comment:
Why are we listing this on an Account and Financial layout? We should have this probably under Financial OR General I would believe.

Reply:
https://github.com/cerner/fhir.cerner.com/commit/62764511c7f719faa5141a6b7bcbfcc839226e9f#r35123720

Reply:
https://github.com/cerner/fhir.cerner.com/commit/e2c16b32dcfacbc09c4d7405900bfeb259552714#diff-d9533e467d767a251d98faa3f8b359bdL1

+1